koku/masu/test/providers/test_report_stats_db_accessor.py

```diff
@@ -134,29 +120,21 @@ def test_update(self):
         self.assertEqual(saver.get_etag(), 'myetag')
 
         saver.delete()
-        saver.commit()
         returned_obj = saver._get_db_obj_query()
         self.assertIsNone(returned_obj.first())
-        saver.close_session()
 
     def test_log_last_started_datetime(self):
         """Test convience function for last started processing time."""
         saver = ReportStatsDBAccessor('myreport', self.manifest_id)
         saver.log_last_started_datetime()
-        saver.commit()
 
         # FIXME: missing asserts
         saver.delete()
-        saver.commit()
-        saver.close_session()
 
     def test_log_last_completed_datetime(self):
         """Test convience function for last completed processing time."""
         saver = ReportStatsDBAccessor('myreport', self.manifest_id)
         saver.log_last_completed_datetime()
-        saver.commit()
 
         # FIXME: missing asserts
         saver.delete()
```

Comment:
Are these tests doing anything?

Reply:
Discussed offline, and updated these tests to have asserts.